web/controllers/account_controller.ex

```diff
@@ -7,20 +7,26 @@ defmodule ResearchResource.AccountController do
   plug :authenticate_user when action in [:index]
 
   def index(conn, _params) do
-    user_details =
+    user_data =
       conn.assigns.current_user.ttrrid
       |> @redcap_api.get_user_data()
-      # Turn user data from a map to a keyword list
-      |> Enum.to_list
-      |> Enum.map(fn {key, val} -> {String.to_atom(key), val} end)
 
-    render conn, "index.html", user_details: user_details
+    cond do
+      user_data ->
+        user_details =
```

Comment:
I use user_details a bit later in the render function, do you think of a better way to pass the value to the view?